test/objects.js

```diff
@@ -681,6 +681,67 @@
     assert.ok(_.isBoolean(false), 'and so is false');
   });
 
+  QUnit.test('isMap', function(assert) {
+    var keyString = 'a string', keyObj = {}, keyFunc = function() {};
+    var obj = new Map([[keyString, 'value one'], [keyObj, 'value two'],
+      [keyFunc, 'value three']]);
+
```

Comment:
You should also avoid populating the `Map` with the constructor+array syntax as that's not supported in IE11.
Use `map.set(key, value)` instead.